src/import_edf.jl

```diff
@@ -204,22 +229,53 @@ can be either a `String` giving the generated channel name, or a `Pair` mapping
 a canonical name to a list of alternatives that it should be substituted for
 (e.g., `"canonical_name" => ["alt1", "alt2", ...]`).
 
+`unit_alternatives` lists standardized unit names and alternatives that map to them.
+See `OndaEDF.STANDARD_UNITS` for defaults.
+
+`preprocess_labels(label::String, transducer_type::String)` is applied to raw edf signal header labels
+beforehand; defaults to returning `label`.
+
 See `OndaEDF.STANDARD_LABELS` for the labels (`signal_names => channel_names`
 `Pair`s) that are used to extract EDF signals by default.
 
 """
-function extract_channels_by_label(edf::EDF.File, signal_names, channel_names)
+function extract_channels_by_label(edf::EDF.File, signal_names, channel_names;
+                                   unit_alternatives=STANDARD_UNITS, preprocess_labels=(l,t) -> l)
     matcher = x -> begin
         # yo I heard you like closures
-        return s -> match_edf_label(s.header.label, signal_names,
-                                    x isa Pair ? first(x) : x,
+        # x is either a channel name (string), or a channel_name => alternatives Pair.
+        this_channel_name = x isa Pair ? first(x) : x
+        return s -> match_edf_label(preprocess_labels(s.header.label, s.header.transducer_type),
+                                    signal_names,
+                                    this_channel_name,
                                     channel_names)
     end
-    edf_channel_names, edf_channels = extract_channels(edf.signals, (matcher(x) for x in channel_names))
-    isempty(edf_channel_names) && return nothing
+    edf_channels = extract_channels(edf.signals, (matcher(x) for x in channel_names))
```

Comment:
Should the `isempty` check remain?

Reply:
Could add a similar bypass back in like so

```
isempty(edf_channels) && return ([], [])
```

but it's not necessary, since that's what already gets returned without the bypass.